docs/getting-started/Typed_Linting.mdx

```diff
@@ -6,7 +6,8 @@ title: Linting with Type Information
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Some typescript-eslint rules utilize the awesome power of TypeScript's type checking APIs to provide much deeper insights into your code.
+Some typescript-eslint rules utilize TypeScript's type checking APIs to provide much deeper insights into your code.
+These rules are slower than traditional lint rules but are much more powerful.
```

Comment:
Just to give a little more insight. "Slower" is a bit vague (how much slower?). The important information is that it's analyzing the whole project, which gives a better idea of what "slower" might mean.

```suggestion
Some typescript-eslint rules utilize TypeScript's type checking APIs to provide much deeper insights into your code. This requires TypeScript to analyze your entire project instead of just the file being linted.

As a result, these rules are slower than traditional lint rules but are much more powerful.
```